nbclient/tests/test_client.py

```diff
@@ -544,6 +544,29 @@ def test_force_raise_errors(self):
             else:
                 assert u"# üñîçø∂é".encode('utf8', 'replace') in str(exc.value)
 
+    def test_reset_kernel_client(self):
+        filename = os.path.join(current_dir, 'files', 'HelloWorld.ipynb')
+
+        with io.open(filename) as f:
+            input_nb = nbformat.read(f, 4)
+
+        executor = NotebookClient(
+            input_nb,
+            resources=self.build_resources(),
+        )
+
+        executor.execute()
+        # we didn't ask to reset the kernel client, a new one must have been created
+        kc = executor.kc
+        self.assertNotEqual(kc, None)
```

Comment:
I guess we're using pytest right? then we can use this?
```suggestion
        assert kc is not None
```

Reply:
Yes.